src/Microsoft.Azure.Relay/Common/InputQueue.cs

```diff
@@ -47,16 +47,9 @@ public int ReadersQueueCount
         }
 
         // Users like ServiceModel can hook this abort ICommunicationObject or handle other non-IDisposable objects
-        public Action<T> DisposeItemCallback
-        {
-            get;
-            set;
-        }
+        public Action<T> DisposeItemCallback { get; set; }
 
-        object ThisLock
```

Comment:
For internal reasons please don't use the `object ThisLock => this.itemQueue` syntax right now. (this code is mirrored in an internal repro.

Reply:
Ok will keep in mind for future contributions :) I'm starting here and want to help as much I can :D